contracts/payment/PaymentSplitter.sol

```diff
@@ -34,50 +38,52 @@ contract PaymentSplitter {
     }
 
     /**
-     * @dev Payable fallback.
+     * @dev The ether received will be logged with `PaymentReceived` events. Note that these events are not fully
+     * reliable: it's possible for a contract to receive ether without triggering this function. This only affects the
+     * reliability of the events, and not the actual splitting of ether.
```

Comment:
I'd mention _how_ this sending can be achieved (selfdestruct and friends).

Reply:
Please see my comment above. https://github.com/OpenZeppelin/openzeppelin-solidity/pull/1713#discussion_r273272500